components/movementsensor/imuwit/imu.go

```diff
@@ -27,17 +29,33 @@ import (
 
 const model = "imu-wit"
 
+var baudRateList = [...]int{115200, 9600}
+
 // AttrConfig is used for converting a witmotion IMU MovementSensor config attributes.
 type AttrConfig struct {
 	Port     string `json:"serial_path"`
 	BaudRate int    `json:"serial_baud_rate,omitempty"`
 }
 
 // Validate ensures all parts of the config are valid.
-func (cfg *AttrConfig) Validate(path string) error {
+func (cfg *AttrConfig) Validate(path string, logger golog.Logger) error {
+	isValid := false
+
+	// Validating serial path
 	if cfg.Port == "" {
 		return utils.NewConfigValidationFieldRequiredError(path, "serial_path")
 	}
+
+	// Validating baud rate
+	for _, val := range baudRateList {
+		if val == cfg.BaudRate {
+			isValid = true
+		}
+	}
+	if !isValid {
+		return errors.Errorf("Baud rate is not in %v", baudRateList)
```

Comment:
please look at how this shows up on the logger and use what `NewConfigValidationFieldRequiredError` is using if it formats in correctly.